modules/acknowledgements/README.md

```diff
@@ -1 +1,7 @@
 The purpose of the Acknowledgement Generator is to aid in dynamically generating an appropriate list of people to acknowledge for any given publication.
+
```

Comment:
Who is the intended user? 
Is there any permission required for this module?
Are there any configuration use by this module and how do they work?